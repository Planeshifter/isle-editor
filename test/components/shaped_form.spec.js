// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import ShapedForm from '@isle-project/components/shaped-form/main.js';


// TESTS //

describe( '<ShapedForm />', function test() {
	it( 'renders a shaped form', () => {
		const form = renderer.create(
			<ShapedForm
				type="star"
				style={{
					background: 'red',
					boxShadow: 'inset 100px 88px 5px yellow'
				}}
			/>
		).toJSON();
		expect( form ).toMatchSnapshot();
	});
});
