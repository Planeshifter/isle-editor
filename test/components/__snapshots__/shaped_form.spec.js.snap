// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShapedForm /> renders a shaped form 1`] = `null`;
